db/migrate/20210225181416_create_category_expert_endorsements.rb

```diff
@@ -0,0 +1,13 @@
+class CreateCategoryExpertEndorsements < ActiveRecord::Migration[6.0]
+  def change
+    create_table :category_expert_endorsements do |t|
+      t.integer :user_id, null: false
+      t.integer :endorsed_user_id, null: false
+      t.integer :category_id, null: false
+    end
+
+    add_index :category_expert_endorsements, [:user_id]
```

Comment:
Perhaps the main index should be `[:user_id, :endorsed_user_id, :category_id], unique:` because nobody can endorse someone for the same thing more than once right? Might as well enforce it in the db.